crates/dekoder/tests/test_load.rs

```diff
@@ -39,19 +39,19 @@ fn destroy() {
 
 #[test]
 fn save_as_other() {
-    let src = v00tar();
+    let src = v015tar();
     let dst = assert_fs::TempDir::new().unwrap();
     // open
     let eko = EKO::extract(src.to_owned(), dst.to_owned()).unwrap();
     // write to somewhere else
-    let tarb = assert_fs::NamedTempFile::new("v0.0b.tar").unwrap();
+    let tarb = assert_fs::NamedTempFile::new("v0.15b.tar").unwrap();
```

Comment:
```suggestion
    let tarb = assert_fs::NamedTempFile::new("write_test.tar").unwrap();
```
this is a temporary name anyways, let's fix this